master/static/migrations/20220914185603_update-allocation-state.tx.up.sql

```diff
@@ -0,0 +1,21 @@
+ALTER TABLE public.allocations
```

Comment:
@stoksc what's the recommended way to update enums? is this okay or is [this approach with temporary state_](https://github.com/determined-ai/determined/blob/master/master/static/migrations/20220609165106_update-job-type-jobs.tx.up.sql#L1) more preferable?

Reply:
I switched to using temporary state to stay consistent